includes/modules/pages/checkout_confirmation/jscript_paypal_incontext.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Javascript enabler for providing and handling the PayPal 
+ *   Express In Context shopping experience.
+ *
+ * @package page
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Sat Aug 13 11:35:35 2016 -0400 Modified in v1.6.0 $
+ */
+ 
+  if (   defined('MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE') && MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE == 'InContext' 
+      && defined('MODULE_PAYMENT_PAYPALWPP_MERCHANTID')    && MODULE_PAYMENT_PAYPALWPP_MERCHANTID    != '' 
+      && defined('MODULE_PAYMENT_PAYPALWPP_STATUS')        && MODULE_PAYMENT_PAYPALWPP_STATUS        == 'True') {
+        
+//        require(DIR_WS_MODULES . 'payment/paypal/paypalwpp_incontext.php');
+if (isset($_SESSION['paypal_ec_token']) && !empty($_SESSION['paypal_ec_token'])
+    && isset($_SESSION['paypal_ec_markflow']) && $_SESSION['paypal_ec_markflow'] = 1) {
+
+}?>
```

Comment:
The conditional code in this file seems unnecessary here. Particularly when there's nothing inside this nested `if`.

... and (unless I'm missing something here despite your explanation) then the outer `if` is repeated in the `require`d file anyway, so feels like it could be excluded and just left to the `require`d file instead.